acceptance/credentials.go

```diff
@@ -26,6 +26,20 @@ var creds = Feature("credentials", "Create a credential set", func(ctx context.C
 		credentialID = cID
 	})
 
+	if credentialType == "multiple" {
```

Comment:
<img width="663" alt="Screen Shot 2019-04-09 at 4 52 45 PM" src="https://user-images.githubusercontent.com/6145422/55834480-0ab41f80-5ae8-11e9-8674-65cdc528ff18.png">

Reply:
Instead of adding this to the default case. I made it its own. My personal preference, feel free to give me your opinion on this.

Reply:
I look forward to seeing the docs on how exactly to pass those tests :-)